script/Ciel/Day 14/14_01B_01_CIEL14B_1G.txt

```diff
@@ -146,6 +146,7 @@ What makes it so terrifying is that it's a natural phenomenon,
 [offset:39180]{
 -- Page 281, Offset 39180.
 -- 「こ、の……！」
+// can this be a swear?
```

Comment:
Ciel doesn't swear

Reply:
Should we change this one then?
https://github.com/Tsukihimates/Tsukihime-Translation/blob/3434928ca29651ebf7576ce28b7c96f572df10ad/script/Ciel/Day%2014/14_01B_02_CIEL14B_2.txt#L78

Reply:
That's the once exception